build/yamls/antrea-ipsec.yml

```diff
@@ -264,6 +264,13 @@ data:
     # CIDR Range for services in cluster. It's required to support egress network policy, should
     # be set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver.
     #serviceCIDR: 10.96.0.0/12
+
+    # Determines how traffic is encapsulated. It has the following options
+    # encap(default): inter-node Pod traffic is always encapsulated and Pod to outbound traffic is masqueraded.
+    # noEncap: inter-node Pod traffic is not encapsulated, but Pod to outbound traffic is masqueraded
+    # hybrid: noEncap if worker Nodes on same subnet, otherwise encap.
+    # noEncapMasq: Pod traffic is not encapsulated nor masqueraded.
```

Comment:
@jianjuns @tnqn 

any input on name change for noEncapMasq ==>
1) passthrough
2)bumpInWire

thx

Reply:
BumpInTheWire is probably more precise in a way. But for people not familiar with the networking term, "passthrough" is probably clearer. I would vote for passthrough, since the term will need to be documented any way, but let's hear what @jianjuns has to say.